stable/hubot/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "hubot.fullname" . }}
+  namespace: {{ .Values.namespace }}
```

Comment:
Again, please remove namespaces from all resources.

Reply:
Sorry, I totally overlooked that. Thanks for reminding 👍